connector/src/main/java/org.apache.flink/connector/nebula/catalog/NebulaCatalog.java

```diff
@@ -93,11 +160,62 @@ public CatalogDatabase getDatabase(String databaseName) throws DatabaseNotExistE
         }
     }
 
+    /**
+     * @param s database name (same as graph space name)
+     * @param catalogDatabase catalog impl
+     * @param b true if contains [if not exists] else false
+     */
+    @Override
+    public void createDatabase(String s, CatalogDatabase catalogDatabase, boolean b)
```

Comment:
please use more meaningful variable name for s and b. 
eg: dataBaseName, useIfNotExists

Reply:
sure, these variable name is so bad. I'll change them. And should i change the variable names in the method it inherits.

In `AbstractNebulaCatalog`, the method definition looks like this:

```java
@Override
public void createDatabase(String s, CatalogDatabase catalogDatabase, boolean b)
        throws CatalogException {
    throw new UnsupportedOperationException();
}
```

Reply:
great catch, the parent method has bad variable names, please change it too, thanks.

Reply:
Sorry for my late reply, this week i am many other things to do, i am glad to do this as soon as possible

Reply:
This [9e2c2b5](https://github.com/vesoft-inc/nebula-flink-connector/pull/66/commits/9e2c2b5cdfd03364af4335eed332bbe132a08588) commit update the bad variable names in `AbstractNebulaCatalog`